### ISSUE TITLE
Bug - Drawer - VO not working with resizable drawer

### ISSUE THREAD

**Issue by tlabaj** [patternfly/patternfly-react#7145]
**Describe the problem**
A clear and concise description of the problem. Which components are affected?
The splitter for the resizable drawer can not be moved with VO.

**How do you reproduce the problem?**
Provide steps to reproduce. A [codesandbox](https://codesandbox.io/s/serverless-cherry-q9t3f ) demonstrating the problem is appreciated.
Use VO on resizable drawer examples

**Expected behavior**
A clear and concise description of the expected behavior.
 you should be able to move the splitter with VO

**Comment by thatblindgeye** [comment]
Based on some conversation that was had in the PR that this spun off from (#7118), and the resources linked below, I'm unsure whether there is a viable fix for this issue (can continue looking into it, though)

- [ARIA APG window splitter pattern](https://www.w3.org/WAI/ARIA/apg/patterns/windowsplitter/) has a list of keyboard interactions, which the resizable drawer does meet in terms of the Left Arrow and Right Arrow
- https://github.com/w3c/aria-practices/issues/130 has a couple of Codepens linked with examples, both of which behave similarly to the PF resizable drawer (VO key + arrow keys doesn't change the splitter size)
- [VoiceOver Keyboard Shortcuts](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) from Deque University lists VO + right or left arrow as reading the next or previous item. When attempting to use VO + arrow keys I assume VO is trying to actually read the next/previous item.
- Our Slider component behaves similarly where VO modifier + arrow keys does not change the slider, and likewise the Aria APG example of a slider matches our behavior

With VoiceOver, users can change what key(s) act as the VO modifier key as well, so if we were to try and implement some logic to ignore the VO modifier we would have to take into account the different modifier key options and also assume doing so wouldn't cause other keyboard handling issues.

@jessiehuff Since you were involved in the original PR, what are your thoughts?

**Comment by jessiehuff** [comment]
So I'm noticing that in retesting, I'm able to use VO+Shift+Down arrow to get into the vertical splitter. Then I'm able to use left and right arrows to move the splitter. I'm wondering if perhaps this isn't an issue now?
